src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -1021,5 +1016,78 @@ public void DisableExportingAudioData()
 
             _audioFrameDecodedCallback = null;
         }
+
+        /// <summary>
+        /// Enables to decode a video data for every frame.
+        /// </summary>
+        /// <remarks><para>The player must be in the <see cref="PlayerState.Idle"/> state.
+        /// And, <see cref="Multimedia.Display"/> must not be set.</para>
+        /// <para>A <see cref="VideoFrameDecoded"/> event is called in a separate thread(not in the main loop).</para>
+        /// <para>The video frame can be retrieved using a <see cref="VideoFrameDecoded"/> event as a media packet.
+        /// So if you change the media packet in the <see cref="VideoFrameDecoded"/> event, it will be displayed on the device.
+        /// The callback function holds the same buffer that will be drawn on the display device.
+        /// and the <see cref="MediaPacket"/> is available until it's destroyed by <see cref="MediaPacket.Dispose()"/>.
+        /// The packet has to be destroyed as quickly as possible after rendering the data
```

Comment:
In above sentence, the subject 'after rendering the data' is 'the packet'. I think it is contextually awkward .

Do you have any reason to use "have to"?  I think developers has to destroy the packet but it is recommended them to destroy the package quickly.

How about the following sentence?
        /// It is recommended to destroy the packet as quickly as possible after the decoded data is rendered on the display.